### ISSUE TITLE
Data Misfit multipliers

### ISSUE THREAD

**Issue by johnweis0480** [simpeg/simpeg#1331]
### Proposed new feature or change:

It seems that many of the codes and directives that set a target misfit do not account for multipliers in front of the misfit term. In my opinion the target misfit should account for the multiplier. If I set dmis-> 2*dmis, the chi target will be wrong.

**Comment by thibaut-kobold** [comment]
Interesting, that is not how I set the joint PGI, with reasons, so this request would break the joint PGI.
For a joint PGI, the multiplier is thought as a way to balance the various data misfits `k0*dmis0 + k1*dmis1 + ...`. But I still want the same target for each `dmis`, independent of the multiplier.

**Comment by thibaut-kobold** [comment]
example here: https://docs.simpeg.xyz/content/tutorials/14-pgi/plot_inv_1_joint_pf_pgi_full_info_tutorial.html#sphx-glr-content-tutorials-14-pgi-plot-inv-1-joint-pf-pgi-full-info-tutorial-py

Especially with the directives
```
# iteratively balance the scaling of the data misfits
scaling_init = directives.ScalingMultipleDataMisfits_ByEig(chi0_ratio=[1.0, 100.0])
scale_schedule = directives.JointScalingSchedule(verbose=True)
```

**Comment by thibaut-kobold** [comment]
if you wish to set dmis-> 2*dmis. I would then strongly prefer that you adjust the standard_deviation of the data instead.

**Comment by johnweis0480** [comment]
@thibaut-kobold Maybe I am misunderstanding you, but if you expect a certain amount of noise for a dataset, and then multiply the misfit, you still want to fit within the same amount of noise correct? That is the behavior I want

**Comment by thibaut-kobold** [comment]
> if you expect a certain amount of noise for a dataset [...] you still want to fit within the same amount of noise
Yes, we agree, that is the behaviour I want too. Maybe I also misunderstood.
For me, that means evaluating `dmis` compared to its target without taking into account the multiplier in front (so that `dmis` or `2 * dmis` does not change my target misfit)
